vector/src/main/java/im/vector/app/features/spaces/SpaceSettingsMenuBottomSheet.kt

```diff
@@ -150,36 +157,25 @@ class SpaceSettingsMenuBottomSheet : VectorBaseBottomSheetDialogFragment<BottomS
         views.leaveSpace.views.bottomSheetActionClickableZone.debouncedClicks {
             val spaceSummary = activeSessionHolder.getSafeActiveSession()?.getRoomSummary(spaceArgs.spaceId)
                     ?: return@debouncedClicks
-            val warningMessage: CharSequence = if (spaceSummary.otherMemberIds.isEmpty()) {
-                span {
-                    +getString(R.string.space_leave_prompt_msg)
-                    +"\n"
-                    span(getString(R.string.space_leave_prompt_msg_only_you)) {
-                        textColor = ContextCompat.getColor(requireContext(), R.color.riotx_destructive_accent)
-                    }
+            val warningMessage: CharSequence? = if (spaceSummary.otherMemberIds.isEmpty()) {
+                span(getString(R.string.space_leave_prompt_msg_only_you)) {
+                    textColor = ContextCompat.getColor(requireContext(), R.color.riotx_destructive_accent)
```

Comment:
I have not seen that first, but you should use the ColorProvider.
We can change that later.

Reply:
rebased and updated